Lib/collections/__init__.py

```diff
@@ -348,12 +349,33 @@ def __getnewargs__(self):
 
 {field_defs}
 """
-
-_repr_template = '{name}=%r'
-
 _field_template = '''\
     {name} = _property(_itemgetter({index:d}), doc='Alias for field number {index:d}')
 '''
+_repr_template = '{name}=%r'
+_new_template = '''
+def __new__(_cls, {arg_list}):
+    'Create new instance of {typename}({arg_list})'
```

Comment:
It may be faster to set the `__doc__` attribute explicitly rather than compile it.

Reply:
Done.